### PR TITLE
`salesforce` close bulk job when finish

### PR DESCRIPTION
**Pull request by mtuchi**
## Details

It turned out that `job.close()` is a promise function and we were calling it without `await`. This led to bulk operation stuck in progress with the status of `Open`. This PR also remove `@openfn/language-template` in `.changeset/config.json`. `@openfn/language-template` is removed on #480 

## Issues

Reference #407 

## Review Checklist

Before merging, the reviewer should check the following items:

- [x] Does the PR do what it claims to do?
- [ ] Are there any unit tests? Should there be?
- [x] Is there a changeset associated with this PR? Should there be? Note that
      dev only changes don't need a changeset.

### CODE REVIEW

File: .tool-versions [hunk 1]
```diff
@@ -1 +1,2 @@
 nodejs 18.17.1
+pnpm 8.6.12
```
Comment (josephjclark) [inline .tool-versions:2]: Does this work? I had trouble getting the pnpm plugin working.

I'll check out the branch and test it out.

Comment (josephjclark) [reply .tool-versions:2]: @mtuchi can we take this out of here and open it up in a new PR?

Looking at the docs we need some better install instructions (ie, add links to the node and pnpm asdf plugins)

It's also a big-ish change and I'd like it merged as a clean PR for the sake of auditing.

Comment (mtuchi) [reply .tool-versions:2]: Certainly

**Review by josephjclark — APPROVED** [review]

**Comment by josephjclark** [comment]
Great @mtuchi - let's bump versions and add release notes, then we can merge!

File: packages/salesforce/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 # @openfn/language-salesforce
 
+## 4.5.2
+
+### Patch Changes
+
+- 2006e88: await `job.close()`
```
Comment (josephjclark) [inline packages/salesforce/CHANGELOG.md:7]: Maybe we can say "fix an issue with bulk jobs not closing"

Also the PR needs to come out of draft so I can merge.

Thanks!

**Pull request merged**